test/core/rebalancingSetToken.spec.ts

```diff
@@ -28,6 +35,10 @@ BigNumberSetup.configure();
 ChaiSetup.configure();
 const { expect } = chai;
 const RebalancingSetToken = artifacts.require('RebalancingSetToken');
+const Core = artifacts.require('CoreMock');
```

Comment:
Make naming consistent? `Core` and `CoreMock`. Maybe go with `CoreMock` so that future we don't trip over this.